torch/_inductor/compile_fx.py

```diff
@@ -270,7 +273,7 @@ def compile_fx_inner(
         graph = GraphLowering(
             gm,
             shape_env=shape_env,
-            num_static_inputs=num_fixed,
+            num_static_inputs=len(static_input_idxs),
```

Comment:
I dont think we're adjusting the static inputs for GraphLowering correctly

Reply:
Good catch! Surprised that none of the tests failed. Pushed a new commit to account for this comment.